qiita_pet/handlers/api_proxy/tests/test_prep_template.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, main
+
+from qiita_pet.handlers.api_proxy.prep_template import prep_template_get_req
+
+
+class TestPrepAPI(TestCase):
+    def test_prep_template_get_req(self):
+        obs = prep_template_get_req(1, 'test@foo.bar')
+        exp = {'18S': [
+            {'id': 1,
+             'name': 'PREP 1 NAME',
```

Comment:
Is there an issue to add the name property to the prep template? I remember that we talked about adding the name property to the artifacts but not sure if there is an issue for the prep template, and I agree that is a good idea.